### PR TITLE
Updates/additions to templates

### LINKED ISSUE #11: General: add templates to render change annex
The change annex exists in all documents including the resource and module docs. 

There needs to be two types of change templates:
- for resource docs, render:
  - change.yaml for the resource schemas (many)
  - change.yaml for the document itself

- For module docs, render:
  - mapping changes
  - change.yaml for the ARM schema (one)
  - change.yaml for the MIM schema (one)
  - change.yaml for the document itself

### PR DESCRIPTION
**Pull request by manuelfuenmayor**
Fixes https://github.com/metanorma/iso-10303-templates/issues/11

### CODE REVIEW

File: resources/module_annex_change_history.adoc [hunk 1]
```diff
@@ -0,0 +1,126 @@
+[[change_history]]
+[appendix,obligation=informative]
+== Change history
+
+[[general]]
+=== General
+
+This annex documents the history of technical modifications made to ISO/TS 10303-42.
```
Comment (ronaldtse) [inline resources/module_annex_change_history.adoc:8]: @manuelfuenmayor this document identifier is probably not for all documents?

Comment (manuelfuenmayor) [reply resources/module_annex_change_history.adoc:8]: @ronaldtse well, it seems like `<<general>>` is not present in any document.
But perhaps we can change that anchor to `[[change_history_general]]`?

Comment (ronaldtse) [reply resources/module_annex_change_history.adoc:8]: Yes please!

**Comment by TRThurman** [comment]
The name of the .adoc file is suspicious.
The terms used in modules and resources are different.
I would have expected module_annex_change_history.adoc to be in the context of modules; and an equivalent .adoc file for resources. If there is only one .adoc file that handles both use cases of modules and resources, it should be so named.

**Comment by ronaldtse** [comment]
@TRThurman I believe @manuelfuenmayor will create a new change history for resources.

**Comment by manuelfuenmayor** [comment]
@ronaldtse @TRThurman sorry. Yes, there is a mistake in the name.
There should be "module_annex_change_history.adoc" for module documents, and "resource_annex_change_history.adoc" for resource documents. I'll fix that.

**Comment by manuelfuenmayor** [comment]
Fixed in https://github.com/metanorma/iso-10303-templates/pull/18

**Pull request merged**